public/_strategies/raindex/1-auction-dca/auction-dca.rain

```diff
@@ -180,6 +180,14 @@ tokens:
     network: polygon
     address: 0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359
     decimals: 6
+  polygon-usdt:
```

Comment:
issue title says usdc but this is usdt

Reply:
Edit, its USDT. The pool is NHT-USDT : 
https://www.dextools.io/app/en/polygon/pair-explorer/0x356a8af4b85cebf8cb6a0331fffc562bce9967d7?t=1734075520601